.travis.yml

```diff
@@ -54,7 +54,7 @@ matrix:
       env: LIBRABBITMQ_VERSION=v0.5.2 TEST_PHP_ARGS=-m
 
 before_install:
-  - sh provision/install_rabbitmq-c.sh ${LIBRABBITMQ_VERSION}
+  - sh -c './provision/install_rabbitmq-c.sh'
```

Comment:
I still think that `${LIBRABBITMQ_VERSION}` was removed by accident. Or did I miss something? 

The initial idea was that we pass desired version to install script and later read it from `$1`.